platform/commonUI/general/res/sass/openmct.scss

```diff
@@ -0,0 +1,39 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+.app-logo {
+    background: url('/platform/commonUI/general/res/images/logo-app.svg') no-repeat center center;
+}
+
+.l-splash-holder {
+    background: #333;
+    .s-splash {
+        border-radius: 10px;
+        box-shadow: 0 5px 50px 25px rgba(255, 255, 255, 0.1);
+    }
+}
+
+.s-splash {
+    background-image: url('/platform/commonUI/general/res/images/bg-splash.jpg');
+    &:after {
+        background-image: url('/platform/commonUI/general/res/images/logo-app-shdw.svg');
+    }
```

Comment:
Ah! I see.

To clarify, the WARP/VISTA dev environments use a strategy for looking up `some/path`:
* First, check plain old `some/path` in the repo
* If that fails, check `node_modules/openmctweb/dist/some/path` in the repo

Their builds do something equivalent (copies from both paths into a single archive). So, it's safe to refer to `platform/commonUI` in either of those environments.

However, if I'm running Open MCT (or WARP, or VISTA) at `http://somehost/some/location/openmct/index.html` (which would generally be the case whenever we're running on the same server as other resources; VISTA runs this way, for instance), and I look for `/platform/commonUI/foo.html`, this will result in the browser requesting `http://somehost/platform/commonUI/foo.html` (outside of the directory where we're installed, probably giving a 404), whereas what you really want in that situation is `http://somehost/some/location/openmct/platform/commonUI/foo.html`